### PR TITLE
Add startupProbe for NL server; Update all the Kubernetes probers

### PR DESCRIPTION
**Pull request by shifucun**
NL server restarts a few times now due to the lack of startup probe.

Removes initialDelaySeconds=10 as it is default value.

### CODE REVIEW

File: deploy/base/deployment.yaml [hunk 1]
```diff
@@ -66,13 +66,15 @@ spec:
               path: /healthz
               port: 8080
             failureThreshold: 30
-            periodSeconds: 10
```
Comment (Fructokinase) [inline deploy/base/deployment.yaml:69]: Is this deleted because the default is 10?

File: deploy/base/deployment.yaml [hunk 2]
```diff
@@ -190,17 +189,19 @@ spec:
           volumeMounts:
             - name: model-config
               mountPath: /datacommons/model
+          startupProbe:
+            httpGet:
+              path: /healthz
```
Comment (Fructokinase) [inline deploy/base/deployment.yaml:194]: Just double checking, I'm assuming /healthz exists in the nl server.

**Review by Fructokinase — APPROVED** [review]

File: deploy/base/deployment.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/base/deployment.yaml:69]: Yes, default is 10

File: deploy/base/deployment.yaml (hunk 2 above)
Comment (shifucun) [reply deploy/base/deployment.yaml:194]: Yes, it exists already

**Pull request merged**